source/views/transportation/other-modes.js

```diff
@@ -25,6 +27,9 @@ const Container = glamorous.view({
   borderColor: c.iosLightBackground,
 })
 
+const GITHUB_URL =
```

Comment:
Ditto to my comment about adding a central constant for this.  Also, does the body of this const need to be on a different line?

Reply:
Say "hi" to prettier – the string + assignment is over 80 chars long.